ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+        const breadcrumbs = this,
+            chart = this.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = chart.options.breadcrumbs,
+            lang = chart.options.lang;
+        let arrow = '';
+
+        if (breadcrumbsOptions && breadcrumb && breadcrumb[1]) {
+            if (breadcrumbsOptions && breadcrumbsOptions.showOnlyLast) {
+                arrow = '◁ ';
+            }
+
+            const button = chart.renderer.button(
+                breadcrumbsOptions.formatter ? breadcrumbsOptions.formatter(breadcrumb, breadcrumbs) : void 0 ||
+                    format(arrow + breadcrumbsOptions.format as string,
+                        { value: breadcrumb[1].name || (lang && lang.mainBreadCrumb) }, chart),
+                posX,
+                posY,
+                function (e: (Event|Highcharts.Dictionary<any>)): void {
+                    // extract events from button object and call
+                    const buttonEvents = breadcrumbsOptions.events && breadcrumbsOptions.events.click;
+                    let callDefaultEvent;
+
+                    if (buttonEvents) {
+                        callDefaultEvent = buttonEvents.call(breadcrumbsOptions, e as any, breadcrumbs);
+                    }
+
+                    // Prevent from click on the current level
+                    if (callDefaultEvent !== false &&
+                        breadcrumbsList[breadcrumbsList.length - 1][1].name !== button.textStr &&
+                        !breadcrumbsOptions.showOnlyLast) {
+                        breadcrumbs.multipleDrillUp(breadcrumb[0]);
+                    }
+                    if (callDefaultEvent !== false &&
+                        breadcrumbsOptions.showOnlyLast) {
+                        breadcrumbs.multipleDrillUp(null as any);
+                    }
+                })
+                .attr({
+                    padding: 3
+                })
+                .css(breadcrumbsOptions.style as any)
+                .addClass('highcharts-breadcrumbs-button')
+                .add(breadcrumbs.breadcrumbsGroup);
+            return button;
+        }
+    }
+
+    /**
+    * Render the separator.
+    *
+    * @requires  modules/breadcrums
+    *
+    * @function Highcharts.Breadcrumbs#renderSeparator
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Breadcrumbs} posX
+    *        Initial horizontal position
+    * @param {Highcharts.Breadcrumbs} posY
+    *        Initial vertical position
+    * @return {SVGElement|void}
+    *        Returns the SVG button
+    */
+    public renderSeparator(this: Breadcrumbs, posX: number, posY: number): SVGElement|undefined {
+        const breadcrumbs = this,
+            chart = this.chart,
+            breadcrumbsOptions = chart.options.breadcrumbs,
+            size = 10;
```

Comment:
Should we somehow connect size and font-size? Or make this configurable, so the change could be like this:

```js
separator: 'triangle'
```
->
```js
separator: {
  symbol: 'triangle',
  size: 10
  ... // place for more options in the future, maybe styles/attributes?
}
```